lms/djangoapps/commerce/utils.py

```diff
@@ -44,6 +44,13 @@ class EcommerceService(object):
     def __init__(self):
         self.config = CommerceConfiguration.current()
 
+    def toggle_commerce_config(self, enabled=False, checkout_page='/test_basket'):
```

Comment:
@rlucioni I should probably move this method to the test_utils.py as it is only used for testing.